Makefile

```diff
@@ -295,20 +298,7 @@ release: clean-release  ## Builds and push container images using the latest git
 
 .PHONY: release-manifests
 release-manifests: $(RELEASE_DIR) ## Builds the manifests to publish with a release
-	kustomize build config > $(RELEASE_DIR)/infrastructure-components.yaml
-
-.PHONY: release-binary
```

Comment:
@sbueringer Was curious to know why do we not need `release-binary` in `release`

Reply:
Not sure why it was added. release-binary is used in CAPA to build clusterawsadm. But we don't have a binary like this for now, so we don't need the target.

Reply:
Perfect. Thanks